recipe/build.sh

```diff
@@ -35,3 +35,6 @@ cmake \
   ..
 make install -j${CPU_COUNT}
 DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib ctest --output-on-failure -j${CPU_COUNT}
+
+mkdir ${PREFIX}/lib/CGAL/cmake-extra
+mv ${PREFIX}/lib/cmake/CGAL/Find*.cmake ${PREFIX}/lib/CGAL/cmake-extra
```

Comment:
This will break `find_package(CGAL)`, because `CGALConfig.cmake` will no longer be found automatically.

Reply:
It's just moving the `Find*.cmake` package, but now CGAL relies only on `CGALConfig.cmake` (there's no more `FindCGAL.cmake`), and it will still be there.

It will only break CGAL if its `find_package()` relies on the non-standard Find modules, but it seems it doesn't.

Reply:
Of of the first think done by `CGALConfig.cmake` is a search of GMP, and it uses our `FindGMP.cmake` for that.

Reply:
Hmm, this is not happening here with the _installed_ version of `CGALConfig.cmake`, it alrady comes with:

```cmake
if (NOT CGAL_IGNORE_PRECONFIGURED_GMP)
  set( GMP_FOUND           "TRUE")
  set( GMP_USE_FILE        "" )
  set( GMP_INCLUDE_DIR     "C:/.../Library/include" )
  set( GMP_LIBRARIES       "C:/.../Library/lib/gmp.lib" )
  set( GMP_DEFINITIONS     "" )
endif()
```

Anyway, it doesn't matter much here, this PR is already closed and I managed to make a local fix (perhaps it would be too hackish to make this proposed fix here anyway).  A definitive fix in CGAL 4.13 would be good already, many thanks!